src/dataStructures.ts

```diff
@@ -0,0 +1,80 @@
+/* -----------------------------------------------------------------------
+                    Virtual Field Environment Software
+   -----------------------------------------------------------------------
+    Instantiation of the objects that will be used for interacting with 
+    Photo Sphere Viewer to build a virtual geology field guide. 
+    Order listed from big-picture environment collection 
+    to micro-environments and associated elements.
+   ----------------------------------------------------------------------- */
+
+// Virtual Field Environment: the total collection of photo sphere environments
+export interface VFE {
+  name: string;
+  map: NavMap;
+  photospere: Photosphere[];
+}
+
+// Navigation map: a birdseye view of the various hotspots within a single 360-environment
+export interface NavMap {
+  src: string;
+  hotspot: HotSpot3d[];
+}
+
+// Photosphere: a single 360-environment
+export interface Photosphere {
+  id: string;
+  src: string;
+  hotspot: HotSpot3d[];
+  backgroundAudio: string;
+}
+
+// hotspot: a clickable resource that is 2d (x, y)
+export interface HotSpot2d {
+  x: number;
+  y: number;
+  toolTip: string;
+  hotspot: HotSpot2d;
```

Comment:
Shouldn't this line be something like `data: HotSpotData` instead?
Sames goes for the other hotspot type.

Reply:
I'm not quite sure how we will be using these objects it seems lol.  Fixing now!

Reply:
Oh, I get it.  Similarly, to how we added hotspot markers.